tools/BaseRunner.py

```diff
@@ -34,3 +44,13 @@ def run(self, tmp_dir, params):
         log, _ = proc.communicate()
 
         return (log.decode('utf-8'), proc.returncode)
+
+    def can_run(self):
+        """Check if runner can be used
+        This method is called by the main runner script (tools/runner) as
+        a sanity check to verify that tool used by the runner is properly
+        installed.
+
+        Returns True when tool is installed and can be used, False otherwise.
+        """
+        return True if shutil.which(self.executable) is not None else False
```

Comment:
Can't this be formulated more readably as
```
  return shutil.which(self.executable) is not None
```
We return a boolean, so not need to convert a boolean to a boolean :)

Reply:
Thanks for pointing that out, it looks better this way